pkg/manifest/v1/alerts.go

```diff
@@ -68,6 +69,7 @@ type AlertNotificationTargetSpec struct {
 // AlertPolicy is a policy for sending alerts.
 type AlertPolicy struct {
 	ObjectHeader `yaml:",inline"`
+	Metadata              Metadata        `yaml:"metadata" validate:"required"`
```

Comment:
What's the purpose of adding `Metadata` here, and in a few other places, for the v1 OpenSLO spec? The `Metadata` is embedded within the `ObjectHeader` type and should be present

Reply:
Yeah, you correctly had the `ObjectHeader` in there.  I had only put the `Metadata` object in there, so this conflict came when I merged from main. Fixed now

Reply:
Thanks!